### ISSUE TITLE
Links usually result in infinite-wait

### ISSUE THREAD

**Issue by sandhawke** [gitpitch/gitpitch#71]
When I have links in my slides, just as bare http://whatever, or markdown, or a-href, and I click on them during the slideshow, it begins the gitpitch-is-working animation and just stays there.

Once, I saw it actually load the page, framed.     But generally, no.   (And I'm on a fast machine on a fast network.)

Seen on  Firefox/UIbuntu 17.04 (Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:53.0) Gecko/20100101 Firefox/53.0) and Chromium (58.0.3029.110 (Developer Build) Built on Ubuntu , running on Ubuntu 17.04 (64-bit))

Otherwise, this is awesome!      Slides are https://gitpitch.com/sandhawke/mastodon-talk-201705 (but currently all the links are made not links for my users).   A broken version is https://github.com/sandhawke/mastodon-talk-201705/commit/407447e8f9e63b16970cc20c7c656ebf43b2d95d

**Comment by gitpitch** [comment]
Hi @sandhawke thanks for the feedback. Apologies for the links issue. The good news is that there is a very simple solution. You need to use a HTML anchor identifying `target="_blank"`, for example:

```
<a target="_blank" href="https://hexdocs.pm/gen_metrics/GenMetrics.html">The Hex Docs</a>
```

The reason is simple and I definitely need to spell this out on the Wiki. Every GitPitch presentation is rendered within an iframe within the GitPitch landing page. So the only way to redirect on a link is to specify the `_blank` target to ensure the link will be opened. Otherwise you get that sense of spinning/going nowhere.

Here is a link to a sample presentation that demonstrates links working properly:

[https://gitpitch.com/onetapbeyond/gen_metrics](https://gitpitch.com/onetapbeyond/gen_metrics)

You should be able to use this same approach to get the links in your own presentation too. Please let me know if you have any problems once you've had a chance to try this out.

**Comment by sandhawke** [comment]
Thanks, yeah, I didn't think to try target="_blank"

But...  in general I'm okay with it being displayed in the iframe, as long as it's displayed.  I guess the problem is that gitpitch is thinking anything I link to needs to be converted from gitpitch html to slides?  

That's challenging.

Best dirty hack I can think of: only assume you're supposed to transform .md URLs.  Have some flag in the HTML for overriding the default either way, so you can make a non ".md" file be rendered, or an .md file be non-rendered.     HTML comment maybe?   

Oh well, yeah, I can use _blank for now.

**Comment by gitpitch** [comment]
Absolutely. I know I need to update the preprocessing on the markdown. When I do this should become a non issue, markdown links will just-work as expected. I'll try get to this soon. Until then, the only workaround I can give you is `_blank` when using an anchor.

**Comment by sandhawke** [comment]
Understood.   I used that, and now the slides work fine.   (Of course, my presentation is already over, but it's possible folks will look at the slides later, so I fixed them.)

**Comment by gitpitch** [comment]
Issue resolved by 580b972d6d8fa4cdcc9ac7a4b03be6cc96573290. Using explicit HTML anchors as a workaround is no longer required. GitPitch presentation markdown can now use regular Markdown link syntax and those links with automatically open into a new tab/window when clicked/tapped.

**Comment by gitpitch** [comment]
Support for implicit Markdown links has just been added. And implicit Markdown link is simply a HTTP url that appears at the start of any line in the `PITCHME.md` file without any further syntax or annotation, for example:

```
https://gitpitch.com
```

Implicit Markdown links will automatically open into a new tab/window when clicked/tapped.
